test/util/test_runner.py

```diff
@@ -74,6 +74,11 @@ def bctest(testDir, testObj, buildenv):
     """
     # Get the exec names and arguments
     execprog = os.path.join(buildenv["BUILDDIR"], "src", testObj["exec"] + buildenv["EXEEXT"])
+    if testObj["exec"] == "./bitcoin-util":
```

Comment:
Shouldn't this (and similar for the other) append `EXEEXT` to `./bitcoin-util` here?

Reply:
I guess not. This line is about parsing of the [`test/util/data/bitcoin-util-test.json`](https://github.com/bitcoin/bitcoin/blob/master/test/util/data/bitcoin-util-test.json) file.

The correct file extension has already been added in the `execprog` variable above, or provided in an environment variable.